tasks/internal_reconcile_report_phantom/schemas/output.json

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/internal_reconcile_report_phantom/schemas/output.json",
+    "title": "internal_reconcile_report_phantom Lambda Output",
+    "description": "The output for the internal_reconcile_report_phantom Lambda.",
+    "type": "object",
```

Comment:
This only returns one object at a time.

Reply:
so this is expected to be an array instead of object?

Reply:
Yes. Each report can identify multiple phantom entries.